files/ru/web/javascript/reference/statements/throw/index.html

```diff
@@ -42,20 +42,30 @@ <h3 id="Example:_Throw_an_object">Пример: Генерирование об
    this.name = "Исключение, определённое пользователем";
 }
 function getMonthName(mo) {
-   mo = mo-1; // Нужно скорректировать номер месяца согласно индексам массива (1=Jan, 12=Dec)
-   var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul",
-      "Aug", "Sep", "Oct", "Nov", "Dec"];
-   if (months[mo] !== undefined) {
-      return months[mo];
-   } else {
-      throw new UserException("Неверно указан номер месяца");
-   }
+   mo = mo - 1; // Нужно скорректировать номер месяца согласно индексам массива (1 = Jan, 12 = Dec)
+   const months = [
+      "Jan",
+      "Feb",
+      "Mar",
+      "Apr",
+      "May",
+      "Jun",
+      "Jul",
+      "Aug",
+      "Sep",
+      "Oct",
+      "Nov",
+      "Dec"
+   ];
+   if (months[mo] !== undefined) return months[mo];
+   
+   throw new UserException("Неверно указан номер месяца");
 }
 
 try {
    // statements to try
-   var myMonth = 15; // 15 находится вне границ массива, что приведёт к исключению
-   var monthName = getMonthName(myMonth);
+   const myMonth = 15; // 15 находится вне границ массива, что приведёт к исключению
+   let monthName = getMonthName(myMonth);
```

Comment:
`let` has code block scope, so it will not work

```suggestion
   monthName = getMonthName(myMonth);
```